_docs/concepts/policies-and-telemetry/config.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Configuration
+description: An overview of the key concepts used to configure Istio's policy enforcement and telemetry collection features.
+weight: 30
+redirect_from:
+    - /docs/concepts/policy-and-control/mixer-config.html
+    - /docs/concepts/policy-and-control/attributes.html
+---
+{% include home.html %}
+
+Istio's policy and telemetry features are configured through a common model designed to
+put operators in control of every aspect of authorization policy and telemetry collection.
+Specific focus was given to keeping the model as simple and possible, while being powerful
+enough to control Istio's many features at scale.
+
+## Attributes
+
+Attributes are an essential concept to Istio's policy and telemetry functionality.
+An attribute is a small bit of data that describes a single property of a specific
+service request or the environment for the request. For example, an attribute can
+specify the size of a specific request, the response code for an operation, the IP
+address where a request came from, etc.
+
+Each attribute has a name and a type. The type defines the kind of data that the attribute holds. For
+example, an attribute can have a `STRING` type which means it has a textual value, or it can have an `INT64`
+type indicating it has a 64 bit integer value.
+
+Here are some example attributes with their associated values:
+
+```plain
+request.path: xyz/abc
+request.size: 234
+request.time: 12:34:56.789 04/17/2017
+source.ip: 192.168.0.1
+destination.service: example
+```
+
+Mixer is the Istio component that implements policy and telemetry functionality.
```

Comment:
Do we want to mention the two services here (istio-policy and istio-telemetry) ?